frontend/src/pages/Students.vue

```diff
@@ -0,0 +1,326 @@
+                      <br> 
+                  </template>
+                  </b-table-column>
+              </template>
+
+
+
+              <!-- status column --> 
+
+              <template v-for="(column, index) in columns" v-if="index == 3">
+                <b-table-column :key="column.id" v-bind="column" sortable>
+                  <template
+                    v-if="column.searchable"
+                    slot="searchable"
+                    slot-scope="props">
+                      <b-input
+                          v-model="props.filters[props.column.field]"
+                          icon="magnify"
+                          size="is-small" />
+                  </template>
+                  <template v-slot="props">
+                      <span v-if="props.row.StatusID == 1" :class="
+                              [
+                                  'tag is-info',
+                              ]">
+                          <li>Screened</li>
+                      </span> 
+
+                      <span v-if="props.row.StatusID == 2" :class="
+                              [
+                                  'tag is-success',
+                              ]">
+                          <li>Matched</li>
+                      </span> 
+
+                      <span v-if="props.row.StatusID == 3" :class="
+                              [
+                                  'tag is-danger',
+                              ]">
+                          <li>Dropped Out</li>
+                      </span> 
+
+                      <span v-if="props.row.StatusID == 4" :class="
+                              [
+                                  'tag is-warning',
+                              ]">
+                          <li>Unmatched</li>
+                      </span> 
+                  </template>
+
+
+
+                  </b-table-column>
+              </template>
+
+        
+
+            <!-- phone column --> 
+
+              <template v-for="(column, index) in columns" v-if="index == 4">
+                <b-table-column :key="column.id" v-bind="column" sortable>
+                  <template
+                    v-if="column.searchable"
+                    slot="searchable"
+                    slot-scope="props">
+                      <b-input
+                          v-model="props.filters[props.column.field]"
+                          icon="magnify"
+                          size="is-small" />
+                  </template>
+                  <template v-slot="props">
+                      {{ props.row.PhoneNumber }}
+                  </template>
+                  </b-table-column>
+              </template>
+              
+          </b-table>
+      </section>
+    </div>
+  </Page>      
+</template>
+
+
+<script>
+
+// Placeholder for API - Note that this will only work after the FullName method has been merged in the backend//
+const API_URL = "http://localhost:3001/api/students";
```

Comment:
Now that we have the dev server proxy, our API_URL should just be `/api/students`.